bundles/org.openhab.binding.somfymylink/src/main/java/org/openhab/binding/somfymylink/internal/model/SomfyMyLinkCommandParamsBase.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.somfymylink.internal.model;
+
+/**
+ * @author Chris Johnson - Initial contribution
+ */
+public class SomfyMyLinkCommandParamsBase {
+
+    public SomfyMyLinkCommandParamsBase(String auth) {
+        this.auth = auth;
+    }
+
+    String auth;
```

Comment:
Please make sure that fields are placed before the constructor.
Also, for these kinds DTOs it might look better to make these fields public and final.

Reply:
done